<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseEditor</class>
 <widget class="QWidget" name="DatabaseEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor of internal mod database</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QTableView" name="databaseView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showAllModsCheckBox">
       <property name="toolTip">
        <string>Display all mods that have been saved in the database.
If unchecked, only installed mods are displayed.</string>
       </property>
       <property name="text">
        <string>Show all mods</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightMainVerticalLayout">
     <item>
      <widget class="QLabel" name="modNameLabel">
       <property name="text">
        <string>Mod name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="modNameLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="modFolderLabel">
       <property name="text">
        <string>Mod folder name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="modFolderNameLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="steamModNameLabel">
       <property name="text">
        <string>Steam mod name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="steamModNameLineEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveModInfoPushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="removeModHorizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="removeModHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="removeModPushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove mod from database</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
